app/project/[id]/page.tsx

```diff
@@ -46,6 +48,17 @@ const DataCollection = observer(() => {
     }
   }, [id])
 
+  useEffect(() => {}, [currentProject])
+
+  useEffect(() => {
+    console.log('I got a change in the image')
```

Comment:
This is the useEffect that doesnt seem to always be picking up the heroImageId being set - even tho I set it in operations/image.ts and also refetch the project